src/js/media-container.js

```diff
@@ -392,6 +396,90 @@ class MediaContainer extends window.HTMLElement {
     });
   }
 
+  handleKeyupEvent(e) {
+    if (this.contains(e.target) || this.shadowRoot.contains(e.target)) {
+      // ignore space bar and enter
+      if (e.target instanceof MediaChromeButton && [' ', 'Enter'].includes(e.key)) {
+        return;
+      } else if (e.target instanceof MediaChromeRange && ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown'].includes(e.key)) {
+        return;
+      }
+
+      let eventName, currentTimeStr, currentTime, detail, evt;
+      const seekOffset = 10;
```

Comment:
yeah, this makes sense. I think also worth calling it out in https://github.com/muxinc/media-chrome/discussions/257.
This actually brings up an interesting point in that the code is currently in `media-container` and not `media-controller` and an attribute and default value should probably live in `media-controller`?

I guess one way of handling that is that we call the `this.handleKeyupEvent` or whatever method on `this` but only implement it in `media-controller` with an no-op method in `media-container`.

Reply:
yeah that's a good point. These interactions are more about "state management"/"state change requests," so they probably belong in `media-controller`. I also think it would be fine to add a second event handler there for keyup, fwiw (which may make more sense anyway if you do key down + key up pairs similar to the components).

Reply:
added a note here https://github.com/muxinc/media-chrome/discussions/257#discussioncomment-3132937